recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -89,6 +100,16 @@ def distribution_exists_on_channel(binstar_cli, meta, fname, owner, channel='mai
 
 
 def upload_or_check(recipe_dir, owner, channel, variant):
+    # If there is a dedicated upload in the recipe use that one
+    if 'upload_or_check_non_existence.py' in os.listdir(recipe_dir):
+        import importlib.util
+        spec = importlib.util.spec_from_file_location('upload_two',
+                                                      os.path.join(recipe_dir,
+                                                                   'upload_or_check_non_existence.py'))
+        mod = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(mod)
+        return mod.upload_or_check(recipe_dir, owner, channel, variant)
```

Comment:
Wouldn't this go into an endless loop?

Reply:
right let me put a flag in the call.

Reply:
Actually technically not, you just have to *not* have this inspection clause in the recipe version.

Reply:
I was thinking of using the new version of this file to upload the new package, so that we don't have to test in a test feedstock. This feedstock will be its own test. That's how it was designed first.